server/scheduling/scheduler.ts

```diff
@@ -0,0 +1,39 @@
+import { initManagers } from '../db/initDB';
+import { IPost } from '../db/models';
+import mongoose = require('mongoose');
+
+// eslint-disable-next-line no-unused-vars
+const { jobManager, postManager } = initManagers();
+const lt = require('long-timeout');
+
+// run all the tasks which are in the database
+(async() => {
+	(await jobManager.getAllJobs()).forEach((elem) => {
+		if (dateDiff(new Date(), elem.data.date) < 0) {
+			jobManager.deleteJob(elem._id);
```

Comment:
Why do you delete this job? If we missed the job (if the server went down), the post won't be published?